e2e-tests/playwright/e2e/catalog-timestamp.spec.ts

```diff
@@ -60,17 +59,24 @@ test.describe.skip("Test timestamp column on Catalog", () => {
     ]);
   });
 
-  test("Toggle ‘CREATED AT’ to see if the component list can be sorted in ascending/decending order", async () => {
-    const createdAtFirstRow =
-      "table > tbody > tr:nth-child(1) > td:nth-child(8)";
-    //Verify by default Rows are in ascending
-    await expect(page.locator(createdAtFirstRow)).toBeEmpty();
+  test("Toggle 'CREATED AT' to see if the component list can be sorted in ascending/decending order", async () => {
```

Comment:
Many of the tests in the scope of the refactor are currently skipped (I still don't understand the purpose and reasoning of #3839 🤯). Can you please un-skip the tests so we can verify the refactored tests are passing? Maybe give it a re-run to verify they are solid?